jekyll/_cci2/faq.md

```diff
@@ -1,11 +1,23 @@
 ---
 layout: classic-docs
-title: "Frequently Asked Questions"
+title: "Frequently Asked Questions FAQ"
```

Comment:
Good catch - added parens (reason for adding it was to make it show for searches on `faq` via Algolia)